securedrop_client/app.py

```diff
@@ -45,7 +45,7 @@
 
 DEFAULT_LANGUAGE = LanguageCode("en")
 DEFAULT_SDC_HOME = "~/.securedrop_client"
-DESKTOP_FILE_NAME = "org.freedomofthepress.securedrop.client"
+DESKTOP_FILE_NAME = "press.freedom.securedrop.client"
```

Comment:
In principle, free desktop files are supposed to be tied to domains that vendors control.

Oddly, this is the only place I can find with `freedomofthepress` instead of `freedomofpress`, but the github org points to the domain https://freedom.press, so this seems like the correct domain.

This is by far the biggest change in my PRs and perhaps should be split into a distinct PR.

Reply:
This is something we've talked about. (I've brought it up in the past, and I don't remember why we decided to leave it like this. @eloquence Does this ring a bell? (It doesn't seem I opened an issue for this.)

@jsoref Let's take this one to a separate PR. :+1:

Reply:
It doesn't ring a bell; looks like the `org.*` naming scheme was originally introduced without comment in #29.

Reply:
Thanks @eloquence, I'll follow up on this in #1589.